src/DssSpell.sol

```diff
@@ -37,7 +75,37 @@ contract DssSpellAction is DssAction {
     // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/d8c711404e9e17db24b611aa03d45507c1993148/governance/votes/Executive%20vote%20-%20November%2016%2C%202022.md -q -O - 2>/dev/null)"
 
     string public constant override description =
-        "2022-11-16 MakerDAO Executive Spell | Hash: 0xf3023b0a9b3c423edf28bf4cdbd61d287ab29265d462a5fae0378227c153ea7b";
+        "2022-11-30 MakerDAO Executive Spell | Hash: TODO";
+
+    uint256 constant internal RAY = 10 ** 27;
+    uint256 constant internal RAD = 10 ** 45;
+    uint256 constant internal MILLION = 10 ** 6;
+
+    bytes32 constant internal ILK = "DIRECT-COMPV2-DAI";
```

Comment:
This V2 is coming from Compound version or the D3M one? As otherwise the `ilk` might be better something like: `DIRECTV2-COMP-DAI`?

Reply:
Coming from Compound. Same as we did for `DIRECT-AAVEV2-DAI`. I figure we will just replace the old one altogether so no need to version the DIRECT.

Reply:
Ok good!